components/filter/filter.js

```diff
@@ -735,6 +735,12 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 	_handleClear() {
 		const dimension = this._getActiveDimension();
 
+		if (dimension.searchType !== 'none') {
+			this._handleClearSearch(dimension);
```

Comment:
Clicking the "Clear" button within a single button was not clearing the search. I confirmed with Jeff that we would want it to. "Clear All" already clears all the searches within the filter so this makes that consistent.